src/atomate2/common/flows/mpmorph.py

```diff
@@ -0,0 +1,493 @@
+            )
+            flow_jobs.append(convergence_flow)
+
+            # convergence_flow only outputs a structure
+            structure = convergence_flow.output
+
+        self.production_md_maker.name = self.name + " production run"
+        production_run = self.production_md_maker.make(
+            structure=structure, prev_dir=prev_dir
+        )
+        flow_jobs.append(production_run)
+
+        if self.quench_maker:
+            quench_flow = self.quench_maker.make(
+                structure=production_run.output.structure,
+                prev_dir=production_run.output.dir_name,
+            )
+            flow_jobs += [quench_flow]
+
+        return Flow(
+            flow_jobs,
+            output=production_run.output,
+            name=self.name,
+        )
+
+    @classmethod
+    def from_temperature_and_steps(
+        cls,
+        temperature: float,
+        n_steps_convergence: int = 5000,
+        n_steps_production: int = 10000,
+        end_temp: float | None = None,
+        md_maker: Maker = None,
+        quench_maker: FastQuenchMaker | SlowQuenchMaker | None = None,
+    ) -> Self:
+        """
+        Create an MPMorph flow from a temperature and number of steps.
+
+        This is a convenience class constructor. The user need only
+        input the desired temperature and steps for convergence / production
+        MD runs.
+
+        Parameters
+        ----------
+        temperature : float
+            The (starting) temperature
+        n_steps_convergence : int = 5000
+            The number of steps used in MD runs for equilibrating structures.
+        n_steps_production : int = 10000
+            The number of steps used in MD production runs.
+        end_temp : float or None
+            If a float, the temperature to ramp down to in the production run.
+            If None (default), set to `temperature`.
+        base_md_maker : Maker
+            The Maker used to start MD runs.
+        quench_maker :  SlowQuenchMaker or FastQuenchMaker or None
+            SlowQuenchMaker - MDMaker that quenches structure from
+                high to low temperature
+            FastQuenchMaker - DoubleRelaxMaker + Static that
+                "quenches" structure at 0K
+        """
+        raise NotImplementedError
+
+
+@dataclass
+class FastQuenchMaker(Maker):
+    """Fast quench flow from high temperature to 0K structures.
+
+    Quenches a provided structure with a single (or double)
+    relaxation and a static calculation at 0K.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    relax_maker :  Maker
+        Relax Maker
+    relax_maker2 :  Maker or None
+        Relax Maker for a second relaxation; useful for tighter convergence
+    static_maker : Maker
+        Static Maker
+    """
+
+    name: str = "fast quench"
+    relax_maker: Maker = Maker
+    relax_maker2: Maker | None = None
+    static_maker: Maker = Maker
+
```

Comment:
is there any point in having generic `Makers` as defaults? nothing would happen unless you pass in explicit model/DFT `Makers`, no? in which case, maybe better not to set defaults at all?

Reply:
The idea was probably to enforce that these are required, but the workflow itself will fail if `relax_maker` and `static_maker` are unspecified. Changed their defaults to `None`